app/layout.tsx

```diff
@@ -13,8 +13,8 @@ const geistMono = Geist_Mono({
 });
 
 export const metadata: Metadata = {
-  title: "Create Next App",
-  description: "Generated by create next app",
+  title: "PawRide - Uber for Dogs",
```

Comment:
Heckin' pawsome 🐾 title update on line 16, buddy! 'PawRide - Uber for Dogs' is legit creative. 🐶🚗💙